solc_json_parser/parser.py

```diff
@@ -302,8 +305,8 @@ def _get_modifiers(node: Dict) -> List[str]:
         func_kind = node.get('kind')
         state_mutability = node.get('stateMutability')
 
-        signature = _get_signature(name, parameters, func_kind)
-        return_signature = _get_signature("", return_type, func_kind)
+        signature = self.get_signature(name, parameters, func_kind)
```

Comment:
`func_kind` implies different kind of function, maybe we could simply name it as `kind`?